onnx/checker.h

```diff
@@ -11,7 +11,7 @@
 #include <utility>
 
 #include "onnx/defs/schema.h"
-#include "onnx/onnx-data_pb.h"
+#include "onnx/onnx-data.pb.h"
```

Comment:
but the file is called onnx-data_pb.h ... why is the new file name better? If so the filename has also to be renamed?

Reply:
Oh that’s right. @cyyever could you share an explanation?

Reply:
onnx/onnx-data_pb.h includes onnx/onnx-data.pb.h, whose declarations are used in this file.

Reply:
Searching references of "onnx-data_pb.h" in all GitHub projects, I find no one project really uses it and it can be safely removed.

Reply:
SG. Thanks!